homestar-runtime/tests/network.rs

```diff
@@ -53,12 +58,16 @@ fn test_libp2p_listens_on_address_serial() -> Result<()> {
 
     let dead_proc = kill_homestar(homestar_proc, None);
     let stdout = retrieve_output(dead_proc);
-
-    assert!(
-        predicate::str::contains("local node is listening on /ip4/127.0.0.1/tcp/7000/p2p/12D3KooWDpJ7As7BWAwRMfu1VU2WCqNjvq387JEYKDBj4kx6nXTN")
-            .eval(stdout.as_str())
+    let logs_expected = check_lines_for(
+        stdout,
+        vec![
+            "local node is listening",
+            "/ip4/127.0.0.1/tcp/7000/p2p/12D3KooWDpJ7As7BWAwRMfu1VU2WCqNjvq387JEYKDBj4kx6nXTN",
+        ],
     );
 
+    assert!(logs_expected);
```

Comment:
let's make this a debug_assert: https://doc.rust-lang.org/std/macro.debug_assert.html